src/components/area/areaMap.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react'
+import { Map, Marker, ScaleControl, LngLatBoundsLike, MapboxMap } from 'react-map-gl'
+import { AreaType } from '../../js/types'
+import { MAP_STYLES } from '../maps/BaseMap'
+
+interface Mappable {
+  id: string
+  metadata: {
+    lat: number
+    lng: number
+  }
+  areaName: string
+}
+
+interface AreaMapProps {
+  subAreas: Mappable[]
+  area: AreaType
+  focused: string | null
+  selected: string | null
+}
+
+/** get the bounds needed to display this map */
+function getBounds (areas: Mappable[]): [[number, number], [number, number]] | null {
```

Comment:
I think we have a utility that calculate bounding box for a list of geojson.  We should try to re-use it in future PR.